src/Stache/Repositories/EntryRepository.php

```diff
@@ -43,6 +44,17 @@ public function find($id): ?Entry
         return $this->query()->where('id', $id)->first();
     }
 
+    public function findOrFail($id): Entry
+    {
+        $result = $this->query()->where('id', $id)->first();
```

Comment:
Would it make sense just to call the find() method here to avoid the duplication of the same query eg

`if (! $result = $this->find($id)) `